test-utils/runtime/src/lib.rs

```diff
@@ -1358,7 +1358,12 @@ mod tests {
 
 		// Try to allocate 1024k of memory on heap. This is going to fail since it is twice larger
 		// than the heap.
-		let ret = client.runtime_api().vec_with_capacity(&block_id, 1048576);
+		let ret = client.runtime_api().vec_with_capacity_with_context(
+			&block_id,
```

Comment:
Will have a conflict here:
```suggestion
			client.chain_info().best_hash,
```